packages/remix/src/fetchMetaData.ts

```diff
@@ -0,0 +1,53 @@
+import type { FrameFlattened } from "frames.js";
+import { FRAMES_META_TAGS_HEADER } from "frames.js/core";
+import type { MetaFunction } from "@remix-run/node";
+
+type Metadata = ReturnType<MetaFunction>;
+
+/**
+ * Fetches meta tags from your Frames app that can be used in Remix meta() function.
+ *
+ * @example
+ * import { fetchMetaData } from "@frames.js/remix";
+ *
+ * export async function loader({ request }: LoaderFunctionArgs) {
+ *   return {
+ *     metaTags: await fetchMetaData(new URL('/frames', request.url)),
+ *   }
+ * }
+ *
+ * export const meta: MetaFunction<typeof loader> = ({ data }) => {
+ *   return [
+ *    { title: 'My title' },
+ *    ...data.metaTags,
+ *   ];
+ * };
+ *
+ * @param url Full URL of your Frames app
+ */
+export async function fetchMetaData(url: URL | string): Promise<Metadata> {
```

Comment:
Can we call this `fetchMetadata` to match the `Metadata` type?

(also applies to other instances of `MetaData` -> `Metadata`)

Reply:
good point